bin/utils/const.dart

```diff
@@ -0,0 +1,18 @@
+import '../menu/base/single_value_field.dart';
+import 'utils.dart';
+
+class Const {
+    // serial values should be unique 
+  static const nodeIdSerial = "node_id";
+  static const parentIdSerial = "node_p_id";
+  static const childIdSerial = "node_c_id";
+  static const nodeNameSerial = "node_name";
+  static const nodeAdditionalInformationSerial = "node_name";
+
+  static SingleValueField<int> nodeIdField = SingleValueField<int>(name: Const.nodeIdSerial, message: "Enter a node id", cast: Utils.castToInt);
+  static SingleValueField<int> parentIdField = SingleValueField<int>(name: Const.parentIdSerial, message: "Enter a the parent node id", cast: Utils.castToInt);
+  static SingleValueField<int> childIdField = SingleValueField<int>(name: Const.childIdSerial, message: "Enter a the child node id", cast: Utils.castToInt);
+
+  static SingleValueField<String> nodeNameField = SingleValueField<String>(name: Const.nodeNameSerial, message: "Enter a the node name", cast: Utils.castToString);
```

Comment:
Should be constants

Reply:
Can't because the value of abstract class [Field](https://github.com/hashsalt0/graph_app/blob/e37b24c03e11958d9fffa7d27f7b1a7e1432f8b9/bin/menu/base/field.dart) is not a const.